baselines/audioworklet.generated.d.ts

```diff
@@ -154,68 +154,40 @@ declare var CountQueuingStrategy: {
 
 /** An event which takes place in the DOM. */
 interface Event {
-    /**
-     * Returns true or false depending on how event was initialized. True if event goes through its target's ancestors in reverse tree order, and false otherwise.
-     */
+    /** Returns true or false depending on how event was initialized. True if event goes through its target's ancestors in reverse tree order, and false otherwise. */
```

Comment:
Currently single line comments are using both single line and multi line forms and I want to unify it.

I chose single line form since it allows simple `/** @deprecated */` rather than three line deprecated comment. @orta sounds good to you?